### ISSUE TITLE
Custom Disaggregation filter is invalid

### ISSUE THREAD

**Issue by mroe1234** [ntop/ntopng#4099]
verson: ntopng Enterprise M Edition v.4.1.200629

It seems that no matter what value I put in for Traffic Filter, including any of the provided examples.  I always get "Invalid filter" error.  As seen below, I copied and pasted one of the examples.

![image](https://user-images.githubusercontent.com/7173287/86122673-37b27080-baa6-11ea-875a-019cd8d42cbf.png)

**Comment by cardigliano** [comment]
@mroe1234 the same filter is working for me, what OS are you running?

**Comment by mroe1234** [comment]
Debian Buster 10.4
4.19.0-8-amd64 #1 SMP Debian 4.19.98-1 (2020-01-26) x86_64 GNU/Linux

**Comment by mroe1234** [comment]
Any ideas on how to even figure out what's throwing the error?

**Comment by cardigliano** [comment]
@mroe1234 it seems we are not able to reproduce this, any chance you can provide access to this system for debugging?

**Comment by mroe1234** [comment]
best I can do is a zoom session

**Comment by cardigliano** [comment]
That should work, please send a direct email to cardigliano at ntop.org to schedule the call. Thank you.

**Comment by mroe1234** [comment]
I think I tracked down the bug/feature.  I deployed a whole new stack to play around with.  The limitation seems to be I cannot create custom disaggregation filters while I'm disaggregating by VLAN ID.  The thought process here was I wanted to duplicate the vlan id disaggregation prior to switching disaggregation methods, so I don't lose data.  That's when I'm presented with the "invalid filter" error message.  However, if I switch to disaggregation: None (and restart ntopng) I am able to create custom filters.

I guess it would be a nice feature to be able to create custom disaggregation filters while disaggregating other ways.  But at least present a better error message?
